tests/RobotFramework/tasks/E2E_Tutorials/Install.robot

```diff
@@ -0,0 +1,47 @@
+*** Settings ***
+Resource              ../../../../resources/common.resource
+Library               SSHLibrary
+# Library               ThinEdgeIO
+Library               Cumulocity
+Library               Process
+Test Setup           Setup    
+# Suite Teardown         Get Logs
+Suite Teardown        Close Connection
+    
+*** Variables ***
+${REMOTE_FILE}    output.txt
+${LOCAL_FILE}    output1.txt
+
+*** Tasks ***
+
+Install/Update of thinedge curl
+   ${log}   Execute Command    curl -fsSL https://thin-edge.io/install.sh | sh -s
+   
```

Comment:
Basic checks have to be added. e.g. running `tedge --help` or checking the mapper and the agent are installed.